### PR TITLE
Push overhaul productization

### PR DESCRIPTION
**Pull request by jstaffor**
## Motivation
Add references to relevant tickets or a short description about what motivated you do it. (E.g JIRA: https://issues.jboss.org/browse/AEROGEAR-{} AND/OR ISSUE: https://github.com/aerogear/standards/issues/{}) 

## What
Add an short answer for: What was done in this PR? (E.g Don't allow users has access to the feature X.)

## Why
Add an short answer for: Why it was done? (E.g The feature X was deprecated.)

## How
Add an short answer for: How it was done? (E.g By removing this feature from ... OR By removing just the button but not its implementation ... ) 

## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Go to `XX >> YY >> SS`
2. Create a new item `N` with the info `X`
3. Try to edit this item 
4. Check if in the left menu the feature X is not so long present.

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task
- [ ] TODO

## Additional Notes

PS.: Add images and/or .gifs to illustrate what was changed if this pull request modifies the appearance/output of something presented to the users.

### CODE REVIEW

File: modules/ROOT/pages/_partials/keycloak/binding.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+:service-name: Identity Management
```
Comment (pmacko1) [inline modules/ROOT/pages/_partials/keycloak/binding.adoc]: @jstaffor do you want to edit keycloak stuff in this pr as well?

Comment (pmacko1) [reply modules/ROOT/pages/_partials/keycloak/binding.adoc]: oh i get it, this is from my quick fix.

Quick update: ak-47 broke "getting started" build so I had to manually reintroduce these two files (keycloak/binding.adoc, /keycloak/keycloak-setup.adoc)

Comment (pmacko1) [reply modules/ROOT/pages/_partials/keycloak/binding.adoc]: i think if you will make a pr push_overhaul->master, these will not be included, so yeah, no worries

**Comment by jstaffor** [comment]
Peter's content review for Push: https://docs.google.com/document/d/1f8Blie9d7BEMOCOl85_oiLl4tck6QegGcAMcz5gIRyw/edit?usp=sharing_eil&ts=5c4b5fb1

**Comment by jstaffor** [comment]
@finp : 
the files have been developed into 8 files, and 1 assembly - all other docs have been deleted
After reviewing live and the render - it's 90% the same in my estimation
Now, consider that the goal here was to "mod doc" it, it still requires a heading update (Peter has contributed to this: https://docs.google.com/document/d/1f8Blie9d7BEMOCOl85_oiLl4tck6QegGcAMcz5gIRyw/edit?ts=5c4b5fb1)
Using the Push Notifications Mobile Service 2019jan25
Using the Push Notifications Mobile Service 2019jan25
imo - I think that if you are happy with the PR as it is, it's best to merge it into the long-running branch and complete the remaining work in a short running branch
let me know what you think

**Comment by jstaffor** [comment]
Closing this in favor of a new PR. To much work required to undo as a result of the [force push](https://github.com/aerogear/mobile-docs/compare/9b53b4b1b51db9a8d243e6911f666861685b78e6..23e11a0b073fffe6f9ff6690cc4648c7dc9108ad).

**Pull request closed without merging**